test/testHill.jl

```diff
@@ -12,4 +13,7 @@
 
     # Check that these at least run
     effects = getODEparams(params, conc)
+    eff2 = getODEparams(p2, conc)
+    num = numcells(effects[:, 3], g1[1]+g2[1], 100)
```

Comment:
Can you put this into a test set with a separate label? You’re specifically testing that this works with nd2 = 0, which is a different goal than the other testing code here.